content/spin/v2/go-components.md

```diff
@@ -67,6 +65,35 @@ func init() {
 func main() {}
 ```
 
+The Spin HTTP component (written in Go) can be built using the `tingygo` toolchain:
+
+<!-- @selectiveCpy -->
+
+```bash
+$ tinygo build -o main.wasm -target=wasi main.go
+```
+
+Once built, we can run our Spin HTTP component (written in Go) using the Spin up command:
+
+<!-- @selectiveCpy -->
+
+```bash
+$ spin up
+```
+
+The Spin HTTP component (written in Go) can now receive and process incoming requests:
```

Comment:
```suggestion
The Spin HTTP component can now receive and process incoming requests:
```

(and are you comfortable with the multiple repetition of "Spin HTTP component"?  It starts to feel a little awkward to me but maybe that's a matter of taste)